Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -934,9 +932,32 @@ bool Renderer::StartFrameDumpToImage(const FrameDumpConfig& config)
   return true;
 }
 
+void writePPM(std::ostream& os, int width, int height, int stride, const u8* data)
+{
+  using namespace std;
```

Comment:
I think that means they're forbidden in header files, and should generally not be used in any other case either.